docs/whats-new.md

```diff
@@ -1,36 +1,62 @@
 # What's New
 
-## [v0.40.1](https://github.com/project-stacker/stacker/releases/tag/v0.40.1)
+## [v1.0.0](https://github.com/project-stacker/stacker/releases/tag/v1.0.0-rc9)
+
+### Convert a Dockerfile for stacker
+
+- A new [`stacker convert`](reference/stacker_cli.md#stacker-convert) command performs a conversion of a Dockerfile into a stacker.yaml file. During the conversion, some variables from the Dockerfile may be exported to a substitution file.  
+
+    :pencil2: The conversion is a best-effort process and may not be successful in all cases.
+
+### Publish specific layers
+
+- By default, the [`stacker publish`](reference/stacker_cli.md#stacker-publish) command pushes all layers in a stacker.yaml file instead of only the required layers. Using a new command option, `layer <value>`, you can explicitly specify which layers are to be published.  This command option can be specified multiple times, selecting each layer to be included.
```

Comment:
i see that 'layer' is hard coded in the command arg. we should change that and change it here to say "image"

Reply:
Should I keep "layers" in this paragraph until the code is changed?

Reply:
sure, that's fine, unless @rchincha has a reason to leave the option as `--layer`, in which case we'll need to explain it here.

also, when we do change it, let's add the dashes so it reads `--layer`

Reply:
When (if?) we change the argument name, should this section become "### Publish specific images"?  And should all instances of 'layer' be changed then to 'image'?